contract_wrapper/KlerosWrapper.js

```diff
@@ -57,16 +58,26 @@ class KlerosWrapper extends ContractWrapper {
   }
 
   /**
-   * Get dispute by caseId. // FIXME mock
-   * @return object
+   * Get dispute by caseId.
+   * @param contractId contract id of dispute
+   * @param artifact defaults to ArbitrableTransaction
+   * @return truffle-contract object
```

Comment:
I think this function must return a dispute object not a truffle-contract.

Reply:
I was thinking that. I started making a dispute object but realized that it is highly dependent on the type of contract so you wouldn't be able to give much information if it is too general. Perhaps it would be better to move getDisputeById to `./contract_wrapper/ArbitrableTransactionWrapper.js`